lnst/Common/Parameters.py

```diff
@@ -179,6 +180,53 @@ def type_check(self, value):
         return value
 
 
+class NetworkParam(Param):
+    """Network Param
+
+    This parameter is used to specify the IP networks a test will have
+     interfaces on. Using it allows multiple simultaneous LNST sessions on a shared broadcast domain.
+
+     It is used with physical or created logical interfaces (vlans, bonds, etc) have
+     addresses on external networks.
+
+     Things like tunnel recipes generally use it for interfaces in the underlying network.
+
+     For example here is VlanRecipe use of it::
+        vlan0_id = IntParam(default=10)
+        vlan1_id = IntParam(default=20)
+        vlan2_id = IntParam(default=30)
```

Comment:
These IntParam parameters are imo irrelevant.

Reply:
Yeah, looking at it again your probably right.